setup/db/db/schema-4930to41000.sql

```diff
@@ -254,4 +254,7 @@ CREATE TABLE `cloud`.`firewall_rules_dcidrs`(
   UNIQUE KEY `unique_rule_dcidrs` (`firewall_rule_id`, `destination_cidr`),
   KEY `fk_firewall_dcidrs_firewall_rules` (`firewall_rule_id`),
   CONSTRAINT `fk_firewall_dcidrs_firewall_rules` FOREIGN KEY (`firewall_rule_id`) REFERENCES `firewall_rules` (`id`) ON DELETE CASCADE
-)ENGINE=InnoDB DEFAULT CHARSET=utf8;
\ No newline at end of file
+)ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+-- CLOUDSTACK-9975: Allow customizing system VM templates for SSVM, VR, and Console Proxy
```

Comment:
4.10 was already closed, right?
I think you should get the master and add it to 4.10 - 4.11? Or something like that